packages/config/src/ast-utils/astConfigHelpers.ts

```diff
@@ -41,15 +42,18 @@ export interface ASTComponentDefinitionConfig {
  * }
  */
 export function addComponentDefinition (config: ASTComponentDefinitionConfig): t.ObjectProperty {
-  return extractProperty(`
-    const toMerge = {
-      component: {
-        devServer: {
-          framework: ${config.framework ? `'${config.framework}'` : 'undefined'},
-          bundler: '${config.bundler}',
-        },
+  const componentConfig = JSON.stringify({
+    component: {
+      devServer: {
+        framework: config.framework,
+        bundler: config.bundler,
       },
-    }
+      specPattern: config.specPattern,
+    },
+  }, null, 2)
+
+  return extractProperty(`
+    const toMerge = ${componentConfig}
```

Comment:
Hm I would expect this would have changed the indentation. Or do we handle the formatting elsewhere :thinking:

Reply:
Right above is the `JSON.stringigify(..., null, 2)` indents properly.